doc/source/install/upgrade_from_5.5.rst

```diff
@@ -0,0 +1,203 @@
+.. _upgrade_from_5.5:
+
+=================================================
+Upgrading Percona XtraDB Cluster from version 5.5
```

Comment:
I don't think we need upgrade from 5.5 doc. Recommended way to upgrade between the versions is to go 5.5 -> 5.6 -> 5.7. (5.5 -> 5.6 is already described in 5.6 manual).